docs/3.0.0-beta.x/plugins/email.md

```diff
@@ -111,3 +111,42 @@ If you want to create your own provider without publishing it on **npm** you can
 ```
 
 - Finally, run `yarn install` or `npm install` to install your new custom provider.
+
+## Trouble shooting
+
+You receive an `Auth.form.error.email.invalid` error even if the email is valid and exist in your database.
```

Comment:
You received an `Auth.form.error.email.invalid` error even though the email is valid and exists in the database.